homeassistant/components/alarm_control_panel/abode.py

```diff
@@ -7,7 +7,7 @@
 import logging
 
 from homeassistant.components.abode import (
-    AbodeDevice, DATA_ABODE, DEFAULT_NAME, CONF_ATTRIBUTION)
+    AbodeDevice, DOMAIN, DEFAULT_NAME, CONF_ATTRIBUTION)
```

Comment:
You are not allowed to overwrite the `DOMAIN` of a platform. The platform domain is always its base component, eg in this case that's `'alarm_control_panel'`. You can import the abode domain as another name eg:
```py
from homeassistant.components.abode import (..., DOMAIN as ABODE_DOMAIN, ...)
```